apps/frontend/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,83 @@
+import { useCallback, useFormController } from "~/libs/hooks/hooks.js";
+import {
+	type Control,
+	type FieldPath,
+	type FieldValues,
+} from "~/libs/types/types.js";
+
+import styles from "./styles.module.css";
+
+type Properties<T extends FieldValues> = {
+	control: Control<T, null>;
+	label: string;
+	name: FieldPath<T>;
+	onClick?: () => void;
+	options?: { label: string; value: string }[];
```

Comment:
We should already have similar type

Reply:
Need more details about this. Do I need to create `InputProperties` type in "~/libs/types" module then use it on both `Input` and `Checkbox` component?

Reply:
We already have RadioInputOption type. Rename it to `InputOption` and use it for your checkbox component